content/en/blog/2022/end-user-discussion-group-apac/index.md

```diff
@@ -5,50 +5,50 @@ date: 2022-12-06
 author: "[Reese Lee](https://github.com/reese-lee)"
 ---
 
-Since July, end users have been getting together to discuss their OpenTelemetry 
-adoption and implementation practices in a vendor-netural space known as the 
-[Monthly Discussion Groups](/community/end-user/discussion-group/) 
-(also referenced as the End User Discussion Groups). 
+Since July, end users have been getting together to discuss their OpenTelemetry
+adoption and implementation practices in a vendor-netural space known as the
+[Monthly Discussion Groups](/community/end-user/discussion-group/)
+(also referenced as the End User Discussion Groups).
 
-Previously, they were only available in the EMEA and AMER regions; the End User 
-Working Group is pleased to announce that **APAC sessions will now be available, 
-starting in January 2023**! 
+Previously, they were only available in the EMEA and AMER regions; the End User
+Working Group is pleased to announce that **APAC sessions will now be available,
+starting in January 2023**!
 
 ## The what
-As mentioned above, these Discussion Groups are monthly meetings for end users 
-to discuss the challenges they're facing in their OTel implementations and to 
-learn from each other, as well as to talk about the project itself and what is 
-working, what isn't, and what they'd like to see. 
+As mentioned above, these Discussion Groups are monthly meetings for end users
+to discuss the challenges they're facing in their OTel implementations and to
+learn from each other, as well as to talk about the project itself and what is
+working, what isn't, and what they'd like to see.
 
-We encourage you to attend and ask questions to learn how other users have 
-implemented OpenTelemetry in their organizations, resolved common issues, and 
-more. Example topics that have been discussed include tail sampling, collector 
-scaling, and OpAMP; there really is no limit to the topics! We simply ask that 
-you be respectful and abide by the [Chatham House Rule](https://www.chathamhouse.org/about-us/chatham-house-rule#:~:text=The%20Rule%20reads%20as%20follows,other%20participant%2C%20may%20be%20revealed.).
```

Comment:
FYI, I dropped the higlight "hash" from the link.

Reply:
Probably best to review the changes to this file while ignoring whitespace differences.